pbspro.spec

```diff
@@ -36,20 +36,47 @@
 # trademark licensing policies.
 #
 
+%if !%{defined pbs_name}
 %define pbs_name pbspro
-%define pbs_client client
-%define pbs_execution execution
-%define pbs_server server
-%define pbs_version 18.1.0
+%endif
+
+%if !%{defined pbs_version}
```

Comment:
As we are changing this spec file to read the PBS Pro version from the newly created tar.gz file, this change will attract another change in below windows file. Else it may cause build fail to get a version in windows builds.

https://github.com/PBSPro/pbspro/blob/master/win_configure/projects.VS2008/gen_pbs_version.bat#L15-L20

This batch script reads pbs_version from this pbspro.spec file if configure is not run on it.

Reply:
You are correct, and a corresponding change for Windows has been made.